src/core/Icon/Icon.md

```diff
@@ -57,13 +57,14 @@ import { suomifiDesignTokens } from 'suomifi-design-tokens';
 const StyledIcon = styled((props) => <Icon {...props} />)({
   height: '50px',
   width: 'auto',
-  margin: '8px 8px 0 0',
+  margin: '8px 0 0 0',
```

Comment:
This has used absolute pixel value previously, but could / should we use a spacing token?